test/e2e/network/network_policy.go

```diff
@@ -1373,8 +1374,13 @@ var _ = SIGDescribe("NetworkPolicy [LinuxOnly]", func() {
 			if err != nil {
 				framework.ExpectNoError(err, "Error occurred while getting pod status.")
 			}
-
-			podServerCIDR := fmt.Sprintf("%s/32", podServerStatus.Status.PodIP)
+			mask := 32
+			if utilnet.IsIPv6String(podServerStatus.Status.PodIP) {
+				mask = 128
+			}
+			_, podServerSubnet, err := net.ParseCIDR(fmt.Sprintf("%s/%d", podServerStatus.Status.PodIP, mask))
+			framework.ExpectNoError(err, "could not parse pod subnet")
+			podServerCIDR := podServerSubnet.String()
```

Comment:
you don't need to do this in the 32/128 case; in that case we're keeping all the bits of the pod IP, so just `podServerCIDR := fmt.Sprintf("%s/%d", podServerStatus.Status.PodIP, mask)` is fine